debezium-core/src/main/java/io/debezium/pipeline/JmxUtils.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.pipeline;
+
+import java.lang.management.ManagementFactory;
+
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.NotCompliantMBeanException;
+import javax.management.ObjectName;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.CommonConnectorConfig;
+
+public class JmxUtils {
```

Comment:
Without a big refactoring the only way I see is to create a dedicated class for the notification and signal JMX that extends the `Metrics` and instantiate it in the init method of the channel and use it to register and unregister the bean. This requires some changes to the `Metrics` class since the `type` is hard coded to `connector-metrics`. Currently the new beans are of `management` `type`.

What I don't like of this solution is that actually the new bean are not exactly a "metrics".  
@jpechane  WDYT?